package.json

```diff
@@ -39,11 +39,11 @@
     "release-patch": "npm version patch -m '%s'"
   },
   "devDependencies": {
-    "chai": "4.2.0",
+    "chai": "^4.3.4",
     "eslint": "6.8.0",
     "esm": "3.2.25",
-    "mocha": "7.0.0",
-    "nyc": "15.0.0",
+    "mocha": "^9.1.1",
```

Comment:
I was running into errors when trying to run specs until I updated these outdated `devDependencies`.